src/content/docs/concepts/security/permissions.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Permissions
+---
+
+Permissions are descriptions of explicit privileges of commands.
+
+```toml
+[[permission]]
+identifier = "my-identifier"
+description = "This describes the impact and more."
+commands.allow = [
+    "read_file"
+]
+
+[[scope.allow]]
+my-scope = "$HOME/*"
+
+[[scope.deny]]
+my-scope = "$HOME/secret"
+```
+
+It can enable commands to be accessible in the frontend of a Tauri application.
+It can map scopes to commands and defines which commands are enabled.
+Permissions can enable or deny certain commands, define scopes or combine both.
+
+Permissions can be grouped as a set under a new identifier.
+This is called a permission set. This allows you to combine scope related permissions
+with command related permissions. It also allows to group or bundle operating
+specific permissions into more usable sets.
+
+As a plugin developer you can ship multiple, pre-defined, well named permissions
+for all of your exposed commands.
+
+As an application developer you can extend existing plugin permissions or
+define them for your own commands.
+They can be grouped or extended in a set to be re-used or to simplify the main
+configuration files later.
+
+## Permission Identifier
+
+The permissions identifier is used to ensure that permissions can be re-used and have unique names.
+
+:::tip
+
+With **crate-name** we refer to the plugin crate name.
+This is meant as namespacing to reduce likelihood of naming conflicts.
+When referencing permissions of the application itself it is not necessary.
+
+:::
+
+- `<crate-name>:default` Indicates the permission is the default for a plugin or application
+- `<crate-name>:default-<command-name>` Indicates the permission is the default for an individual command
```

Comment:
we don't really use this default-command-name thing anywhere, just `allow-*` and `deny-*`